### PR TITLE
Enable `import/no-anonymous-default-export`

### PR DESCRIPTION
**Pull request by sosukesuzuki**
## Description

ref: https://github.com/import-js/eslint-plugin-import/blob/v2.24.1/docs/rules/no-anonymous-default-export.md#importno-anonymous-default-export

For consistently name between importer and exporter, and improve editor intellisense.

## Checklist

- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: scripts/build/config.mjs [hunk 1]
```diff
@@ -162,4 +162,5 @@ const coreBundles = [
   ...bundle,
 }));
 
-export default [...coreBundles, ...parsers];
+const config = [...coreBundles, ...parsers];
+export default config;
```
Comment (fisker) [inline scripts/build/config.mjs:166]:
```diff
const configs = [...coreBundles, ...parsers];
export default configs;
```

**Comment by fisker** [comment]
Personally, I prefer this rule only check function and classes, but acceptable if you prefer to enfore them all.

File: .eslintrc.js [hunk 2]
```diff
@@ -260,5 +261,11 @@ module.exports = {
         sourceType: "module",
       },
     },
+    {
+      files: ["scripts/build/shims/*.mjs"],
+      rules: {
+        "import/no-anonymous-default-export": "off",
+      },
+    },
```
Comment (fisker) [inline .eslintrc.js:269]: Let's fix them?

**Comment by sosukesuzuki** [comment]
> Personally, I prefer this rule only check function and classes, but acceptable if you prefer to enfore them all.

Thanks, I prefer the style of this PR because of the consistency of naming between exporter and importer.

**Comment by sosukesuzuki** [comment]
Oh I mistook rabasing. I'll fix it.

**Review by fisker — APPROVED** [review]

**Pull request merged**